File: Gfg/Palindrome_number.py
n=int(input("Enter number:")) # take input from user
temp=n
rev=0
while(n>0):
    dig=n%10 #extract digits
    rev=rev*10+dig#reverse digits
    n=n//10
if(temp==rev):
    print("The number is a palindrome!")
else:
    print("The number isn't a palindrome!")

File: Gfg/C/Palindrome Number.c
#include <stdio.h>

int main()
{
	int num,temp=0;
	scanf("%d",&num);   //taking input
	int num2 = num;
	while(num>0)
	{
		temp = (temp*10) + (num%10);
		num = num/10;
	}
	
	if(temp==num2)
	{
		printf("%d is palindrome.\n",num2);
	}
	else
	{
		printf("%d is not palindrome.\n",num2);
	}
	
	
	return 0;
}

